tests/integration/targets/aws_s3/tasks/main.yml

```diff
@@ -273,6 +273,39 @@
         that:
           - result is changed
 
```

Comment:
@na4da Thank you for working on this. Overall it looks good to me. But what's happening if you are trying to get an inexistent object?

Reply:
@na4da Thank you for explaining. Good!